modules/navier_stokes/include/fvkernels/PINSFVMomentumFriction.h

```diff
@@ -0,0 +1,41 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "FVElementalKernel.h"
+
+/**
+ * Imposes a friction force on the momentum equation in porous media.
+ */
+class PINSFVMomentumFriction : public FVElementalKernel
```

Comment:
For some objects, such as this one, I expect there to be no difference between the PCNSFV and PINSFV implementations. In my rebase on your work, I have been renaming objects such as these from `PINSFV\(.*\)` to `PNSFV\1`. I'm fine continuing to do that work myself, but you may also think about doing it upstream in this PR. So far I believe this object, as well as `PNSFVMomentumPressure` (the flux version) and `PNSFVMomentumGravity` are the objects I've renamed

Reply:
I understand there may be some confusion for users if some objects have `PINSFV` prefix and others have `PNSFV` prefix. In that case, I'd suggest maybe we do in the .h file:

```
using PINSFVMomentumFriction = PNSFVMomentumFriction;
```

and in the .C file:
```
registerMooseObject("NavierStokesApp", PINSFVMomentumFriction);
registerMooseObject("NavierStokesApp", PNSFVMomentumFriction);
```

Reply:
ok I ll let you do the renaming

Reply:
We could do that or we could assume users are going to use actions anyway.
How does this do for documentation? Will we have to have two entries and two pages?

Reply:
Yea I think the assumption about actions is fine. I actually have no idea how documentation would work for the way I proposed

Reply:
For a number of INSFV objects that do not change in PINSFV, I have not duplicated them. For example INSFVNaturalFreeSlipBC. I guess we should decide if we want to register PINSFV for everything now or just allow the mixing of PINSFV, INSFV, NSFV depending on what is valid when.